src/languageHandler.tsx

```diff
@@ -73,6 +74,24 @@ export function _td(s: string): string { // eslint-disable-line @typescript-esli
     return s;
 }
 
+/**
+ * to improve screen reader experience translations that are not in the main page language
+ * eg a translation that fell back to english from another language
+ * should be wrapped with an appropriate `lang='en'` attribute
+ * counterpart's `translate` doesn't expose a way to determine if the resulting translation
+ * is in the target locale or a fallback locale
+ * for this reason, we do not set a fallback via `counterpart.setFallbackLocale`
+ * and fallback 'manually' so we can mark fallback strings appropriately
+ * */
+const translateWithFallback = (text: string, options?: object): { translated?: string, isFallback?: boolean } => {
```

Comment:
Why the `isFallback` parameter?

Reply:
We need to mark when translations are in a language != the lang attributed on the page level. The `isFallback` param in the result is used to identify which translations fell back to english so we can wrap them with `<span lang='en'>` in `_tDom`